django_netjsongraph/base/admin.py

```diff
@@ -60,7 +74,7 @@ def get_actions(self, request):
         return actions
 
     def get_extra_context(self, pk=None):
-        ctx = {}
+        ctx = super(AbstractTopologyAdmin, self).get_extra_context(pk)
```

Comment:
@rohithasrk another thing I've noticed is that `submit_line` must be defined also in `add_view` otherwise the add topology view page will break